src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java

```diff
@@ -885,6 +893,9 @@ public void setStringStackSize(int stackSize) {
         wrappedClient.setStringStackSize(stackSize);
     }
 
+    @Override
+    public void setCameraPitchTarget(int cameraPitchTarget) { wrappedClient.setCameraPitchTarget(cameraPitchTarget); }
```

Comment:
Seems like `setCameraYawTarget` is missing

Reply:
it already exists

Reply:
Ah I see, I was comparing with my local changes and I had moved it to be next to `setCameraPitchTarget`